src-plugins/vtkDataMesh/writers/vtkDataMeshWriter.cpp

```diff
@@ -53,8 +59,6 @@ bool vtkDataMeshWriter::write(const QString& path)
   if (!this->data())
     return false;
 
-  qDebug() << "Can write with: " << this->identifier();
-
   medAbstractData * medData = dynamic_cast<medAbstractData*>(this->data());
```

Comment:
Could you tell where you need `this->data()` and you cannot access it through vtkMetaSurfaceMesh?